garrysmod/gamemodes/sandbox/entities/entities/gmod_ghost.lua

```diff
@@ -20,13 +20,15 @@ end
 --[[---------------------------------------------------------
 	Name: SetBonePosition (serverside)
 -----------------------------------------------------------]]
-function ENT:SetNetworkedBonePosition( i, Pos, Angle )
+function ENT:SetNWBonePosition( i, Pos, Angle )
 
-	self:SetNetworkedVector( "Vector" .. i, Pos )
-	self:SetNetworkedAngle( "Angle" .. i, Angle )
+	self:SetNWVector( "Vector" .. i, Pos )
+	self:SetNWAngle( "Angle" .. i, Angle )
 
 end
 
+ENT.SetNetworkedBonePosition = meta.SetNWBonePosition

```

Comment:
Why not? Get/SetNetworkedVarProxy was changed to NW; this is the only "Networked" function left